docs/binary2.md

```diff
@@ -17,42 +17,83 @@ nav_order: 2
 ---
 
 ## Addition
-
+Binary addition is similar to Decimal addition. As this addition is binary implies we cannot have a number 
+greater than 1 i.e., when you do '1+1' it gives 0 with carry 1 i.e, 10.
 ```yaml 
-1. 0 + 0 = 0
-2. 0 + 1 = 1
-3. 1 + 0 = 1
-4. 1 + 1 = 1
```

Comment:
I think the operation depicting simple usage must not be removed

Reply:
I have already removed that, those are the removed lines. Please look closely to what I have changed 😅